packages/app-mobile/root.tsx

```diff
@@ -890,6 +897,37 @@ class AppComponent extends React.Component {
 			</View>
 		);
 	}
+
+	private async navigate(url: string) {
+		if (!url) return;
+		let parsedUrl;
+		try {
+			parsedUrl = new URL(url);
+		} catch (_) {
+			// invalid URL
+			return;
+		}
+		reg.logger().info(`navigate to ${url}`);
+
+		await this.props.dispatch({ type: 'NAV_BACK' });
+		await this.props.dispatch({ type: 'SIDE_MENU_CLOSE' });
+		const { pathname } = parsedUrl;
+		if (pathname.startsWith('//notes/')) {
```

Comment:
That's more like a temporary hack I guess because the urls should be parsed properly? So for now please could you leave a comment mentioning that it should be done via the callbackUrl lib.

Reply:
Removed this logic all together - link logic is now in #5764.